execute-test-cases.mdx

```diff
@@ -79,15 +79,11 @@ Red signals the test has failed. This might have 3 reasons:
 
 ## Schedule regular test runs
 
-The point of software testing is to test regularly. You can [schedule tests](/scheduled-execution.mdx) by clicking the `schedule` button.
+The point of software testing is to test regularly. You can [schedule tests](/scheduled-execution.mdx) by clicking the `schedule` button in `project overview`.
 This is a great strategy for synthetic monitoring of your app in production.
 
 <Frame caption="Scheduling test runs 7/2024">
-  <img
-    src="/images/executing/schedule.png"
-    alt="scheduling test runs"
-    width="300px"
-  />
+  <img src="/images/executing/schedule.png" alt="scheduling test runs" />
```

Comment:
did you test multiple screen sizes to verify it looks ok without the width?

Reply:
yup, looks good to me, it's a new screenshot